### PR TITLE
chore: migrate to owlbot

### PR DESCRIPTION
**Pull request by Neenu1995**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -149,7 +149,32 @@
         <dependencies>
             <!-- Production Deps have been moved to google-cloud-bigtable-deps-bom -->
 
-            <!-- Test Deps in alphabetical order -->
+            <dependency>
+        <groupId>com.google.api.grpc</groupId>
```
Comment (kolea2) [inline pom.xml:153]: why are these getting added?

File: versions.txt [hunk 2]
```diff
@@ -6,4 +6,3 @@ grpc-google-cloud-bigtable-admin-v2:2.1.2:2.1.2
 grpc-google-cloud-bigtable-v2:2.1.2:2.1.2
 proto-google-cloud-bigtable-admin-v2:2.1.2:2.1.2
 proto-google-cloud-bigtable-v2:2.1.2:2.1.2
-google-cloud-bigtable-emulator:0.138.2:0.138.2
```
Comment (kolea2) [inline versions.txt:9]: this shouldn't be removed

**Review by kolea2 — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (Neenu1995) [reply pom.xml:153]: Owlbot adds dependencies for all modules in the parent pom. It's one of the features helping it to propose a new client library version without human intervention.

Does this affect the library build?

File: versions.txt (hunk 2 above)
Comment (Neenu1995) [reply versions.txt]: Looking into this.

File: pom.xml (hunk 1 above)
Comment (chingor13) [reply pom.xml:153]: This is also just the dependency management section, not the actual dependencies

**Pull request closed without merging**